calliope/config/defaults.yaml

```diff
@@ -80,6 +80,7 @@ default_tech:
         storage_cap_equals: false  # name: Specific storage capacity ¦ unit: kWh ¦ If not defined, ``energy_cap_equals`` * ``charge_rate`` will be used as the capacity and overrides ``_max`` and ``_min`` constraints.
         storage_cap_max: .inf  # name: Maximum storage capacity ¦ unit: kWh ¦ If not defined, ``energy_cap_max`` * ``charge_rate`` will be used as the capacity.
         storage_cap_min: 0  # name: Minimum storage capacity ¦ unit: kWh ¦
+        storage_cap_ratio: false # name: Ratio of storage capacity relative to energy capacity ¦ unit: fraction ¦
```

Comment:
I'm not sure this is sufficiently descriptive. Perhaps it should be `energy_cap_per_storage_cap`, or 
`charge_rate` should be changed to include a `min`, `equals`, and `max`?